concepts/review.md

```diff
@@ -1,5 +1,103 @@
-# Work in Progress
+# Getting Analysis Review
+
+The current process for getting analysis review has not been optimized.
+If you run into problems, feel free to ask #datapipeline
+
+## Table of Contents
+
+<!-- toc -->
+
+# Finding a Reviewer
+
+There are two major types of review for most analyses:
+**Data review** and **Methodology review**.
+
+A **data reviewer** has **expert understanding of the dataset** your analysis is based upon.
+During data review, your reviewer will try to identify
+any issues with your analysis that come from misapplying the data.
+For example, sampling issues or data collection oddities.
+
+**Methodology review** primarily covers **statistical concepts**.
+For example, if you're training regressions, forecasting data, or doing complex
+statistical tests, you should probably get Methodology review.
+Many (most?) analyses at Mozilla use simple aggregations and little to no statistical inference.
+These analyses only require data review.
+
+Accordingly, we suggest you **first find a data reviewer for your analysis**.
+Your data reviewer will tell you if you should get methodology review
+and will help you find a reviewer.
+
+## Data Reviewers
+
+To get review for your analysis,
+contact one of the data reviewers listed for the dataset your analysis is based upon:
+
+| Dataset           | Reviewers   |
+| :---              | :---        | 
+| Longitudinal      |             |
+| Cross Sectional   |             |
+| Main Summary      |             |
+| Crash Summary     |             |
+| Crash Aggregate   |             |
+| Events            |             |
+| Sync Summary      |             |
+| Addons            |             |
+| Client Count      |             |
+
+# Jupyter Notebooks
+
+It's difficult to review Jupyter notebooks on Github.
```

Comment:
Jupyter notebooks can be exported as MD files and the github gist function allows a nice view in a separate tab. Normally when reviewing other people's notebooks that's my workflow. It isnt prefect but it allows a pretty comprehensive review using github's existing tooling.